client/Components/Games/NewGame.jsx

```diff
@@ -36,7 +36,8 @@ class NewGame extends React.Component {
             password: '',
             useGameTimeLimit: false,
             hideDecklists: false,
-            gameTimeLimit: 35
+            gameTimeLimit: 35,
+            swap: false
```

Comment:
This shouldn't be necessary if you're not offering an option at game creation, just default it on the server